app/views/orders/show.html.erb

```diff
@@ -1,63 +1,28 @@
-<h1><%= "Order Number: #{@order.id}" %></h1>
+<br><br><h1><%= "Order ##{@order.id}" %></h1>
 
 <div class="row">
-  <div class="col-md-3"><h4>Details:</h4></div>
-  <div class="col-md-6"></div>
-  <div class="col-md-3 status pull-right">
-    <div>
-      <% if current_user.role.eql?("admin") && @order.status == "Ordered" || @order.status == "Paid" %>
-        Change Status:
-        <%= form_for [:admin, @order] do |f| %>
-          <%= f.collection_select(:status, Status.all, :id, :name, { selected: @order.status_id }) %>
-          <%= f.submit "Update Order Status" %>
-        <% end %>
-      <% else %>
-        <strong>Status:</strong>
-        <%= @order.status %> <br/>
-        <%= link_to "Your Orders", orders_path(current_user) %>
-      <% end %>
-    </div>
```

Comment:
we probably still want this functionality for store admins, so how about we keep this code in the app / comment out what isn't working for now and when we get to that section we will fix it up.